test/test_helper.exs

```diff
@@ -1,5 +1,3 @@
 ExUnit.start()
 
 { :ok, _ } = Dummy.Repo.start_link
-{ :ok, _ } = Application.ensure_all_started(:postgrex)
```

Comment:
This is advice of these packages maintainers. I prefer to store them as is until according changes go out of docs.